### PR TITLE
Updated help for Preject

### PR DESCRIPTION
**Pull request by cianoc**
The initializer wasn't documented properly. Also expanded the documentation generally to hopefully be clearer.

### CODE REVIEW

File: HelpSource/Classes/Preject.schelp [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+summary:: Reject values from a pattern.
 categories:: Streams-Patterns-Events>Patterns>Filter
-related:: Classes/Pselect, Classes/Pcollect
+related:: Classes/Pselect, Classes/Pcollect, Classes/Collection#-reject
 
 description::
-Rejects values for which the function returns true. The value is passed to the function.
+Preject filters the source strong::pattern:: using strong::func::. Values for which strong::func:: returns true will not be returned by Preject.
 
+This is the pattern libraries equivalent of link::Classes/Collection#-reject::.
```
Comment (brianlheim) [inline HelpSource/Classes/Preject.schelp]: -> library's

**Comment by cianoc** [comment]
I modified the summary as well as fixing the library's thing. Let me know.

**Comment by cianoc** [comment]
@LFSaw Those functions aren't private so I don't think that would be the right thing to do here.

**Comment by LFSaw** [comment]
see also #3158

**Pull request merged**